pihole

```diff
@@ -54,6 +54,11 @@ flushFunc() {
   exit 0
 }
 
+arpFunc() {
+  "${PI_HOLE_SCRIPT_DIR}"/piholeARPTable.sh "$@"
```

Comment:
Perhaps we can `shift` so that the script is not passed `arpflush` in the arguments.

Reply:
Passing `arpflush` is intentional. We might want to add further commands for this script.